_data/compatibility/wasabi.yaml

```diff
@@ -10,17 +10,17 @@ rbf:
     version: "1.0"
   features:
     receive:
-      notification: "na"
+      notification: "false"
```

Comment:
Notification field being `na` means that the wallet/service does not have notification for incoming transactions.
Notification field being `false` means that the wallet/service does have a notification for incoming transactions but that the notification doesn’t show if the tx is signaling RBF.

That said, I just tested again and I did not get a notification. So I believe `na` is correct for this field.

Reply:
> Notification field being na means that the wallet/service does not have notification for incoming transactions.
> Notification field being false means that the wallet/service does have a notification for incoming transactions but that the notification doesn’t show if the tx is signaling RBF.

The wallet have notifications on OSX and Linux, but not on Windows. Anyhow, it's going to be refactored to in-wallet (non-os-native) notifications for all platforms. At this point however we can think about it as a partially implemented feature and such feature should not be considered implemented, so `na` should work, so I misunderstood it. I change it back. Will update to `false` after the native notifications are implemented.  https://github.com/zkSNACKs/WalletWasabi/pull/2119#issuecomment-521191190